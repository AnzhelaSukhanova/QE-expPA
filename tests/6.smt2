(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(declare-fun z () (_ BitVec 4))
(declare-fun t () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and 
 (or (bvult x (bvmul #b0011 y)) (= x (bvmul #b0011 y))) 
 (or (bvult z x) (= z x)) 
 (or (bvult x t) (= x t)))))
