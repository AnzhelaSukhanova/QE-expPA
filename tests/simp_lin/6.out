(and (not (bvult (bvmul y #b0011) z)) (not (bvult t z)))
